src/com/geekhub/hw2/parts/gear/AutomaticTransmission.java

```diff
@@ -0,0 +1,17 @@
+package com.geekhub.hw2.parts.gear;
+
+import com.geekhub.hw2.Force;
+
+public class AutomaticTransmission implements Gear {
```

Comment:
It's abstraction due to can be CarAutomaticTransmission, BoatAutomaticTransmission BikeAutomaticTransmission etc. 
The same for MechanicTransmission, InternalCombustionEngine and ElectricEngine etc.